db/migrate/20191211203831_remove_unique_facility_code_constraint_weams.rb

```diff
@@ -0,0 +1,8 @@
+class RemoveUniqueFacilityCodeConstraintWeams < ActiveRecord::Migration[5.2]
+  disable_ddl_transaction!
+
+  def change
+    remove_index :weams, [ :facility_code ]
+    add_index :weams, [:facility_code], algorithm: :concurrently
+  end
```

Comment:
I should be able to rollback from this migration.
If I rollback as is, `weams` will not have the `unique` constraint on the index.

Reply:
added down method